app/models/service_template_ansible_tower.rb

```diff
@@ -52,6 +52,10 @@ def self.validate_config_info(options)
     config_info
   end
 
+  def my_zone
+    job_template.manager.try(:my_zone) || MiqServer.my_zone
```

Comment:
You are referring to [service_template_provision_task.rb#L146](https://github.com/ManageIQ/manageiq/blob/master/app/models/service_template_provision_task.rb#L146).

I guess my follow-up question would be: Is the service_template valid if there is no manager or if the manager has a `nil` zone?  I do not think it is, therefore I still see value in simplifying this method to `job_template.manager.try(:my_zone)`.